paper/paper.md

```diff
@@ -0,0 +1,96 @@
+
+While `get_grades()` encapsulates the entire computational functionality of the R package, it is compromised for four sequential functions:
+
+-   `process_gs()` ensures the correct format of the Gradescope csv.
+
+-   `process_policy()` similarly ensures the correct format of the policy file.
+
+-   `reconcile_policy_with_gs()` checks the compatibility of the policy file and the Gradescope data.
+
+-   `calculate_grades()` computes the course grades and returns the final grade (and the scores for every intermediate category) appended to the original Gradescope data.
+
+
+# Comparison to Other Packages
+
+Most other commonly-used packages -- particularly for R packages -- are used for
+grading on an assignment-level basis. For example, the `gradeR` package "helps
+grade your students’s assignment submissions that are R Scripts" [@GradeR] whereas `gradebook` is used for the computations of the final, overall course grade.
+The software that has the most similar computational purpose as `gradebook` are
+popular learning-management systems (LMS) used in higher education. This includes
+Canvas [@canvas_gradebook_guide], Moodle [@moodle_grade_calculations], Blackboard Learn [@blackboard_calculate_grades], and D2L Brightspace [@d2l_about_grades], all of which provide an
+integrated gradebook that allow the instructor to specify the manner in which
+assessment scores are used to calculate a final course grade. What makes Gradebook unique is its flexibility of functionality and its capacity for instructor collaboration and contribution:
+the flexible YAML structure allows for the former and the open-source nature of the project allows for the latter.
+
+# Acknowledgements
+
+The authors would like to thank lain Carmichael, Calvin Carter, and Zach Turner for helpful ideas and discussions throughout the development of this project.
+
+As of summer 2024, we are funded by the RTL grant from the Statistics Departmemt at University of California, Berkeley. 
+
+As of October 2024, there is a pending poster submission to Technical Symposium on Computer Science Education (SIGCSE) that utilizes this package, called "Your Grades Are Wrong - Nemo Gradebook: A tool for easy, accurate course grades". The poster was submitted by the same set of authors.
```

Comment:
Is this explicitly required by JOSS? If not, I'd be inclined to remove it since it's a very time-specific reference (to the date and that it's in submission).